comps/llms/text-generation/bedrock/llm.py

```diff
@@ -0,0 +1,85 @@
+import os
+import boto3
+import json
+from fastapi.responses import StreamingResponse
+from comps import CustomLogger, GeneratedDoc, LLMParamsDoc, ServiceType, opea_microservices, register_microservice
+from typing import Union
+from comps import (
+    CustomLogger,
+    GeneratedDoc,
+    LLMParamsDoc,
+    SearchedDoc,
+    ServiceType,
+    opea_microservices,
+    register_microservice,
+    register_statistics,
+    statistics_dict,
+)
+from comps.cores.proto.api_protocol import ChatCompletionRequest
+
+bedrock_runtime = boto3.client(
+    service_name="bedrock-runtime", region_name="us-east-1")
+
+logger = CustomLogger("llm_bedrock_anthropic")
+logflag = os.getenv("LOGFLAG", True)
+
+
+@register_microservice(
+    name="opea_service@llm_bedrock_anthropic",
+    service_type=ServiceType.LLM,
+    endpoint="/v1/chat/completions",
+    host="0.0.0.0",
+    port=9000,
+)
+def llm_generate(input: Union[LLMParamsDoc, ChatCompletionRequest, SearchedDoc]):
+    if logflag:
+        logger.info(input)
+    print(input.query)
+    kwargs = {
+        "modelId": "us.anthropic.claude-3-5-haiku-20241022-v1:0",
```

Comment:
Consider using an environment variable to set the model ID to facilitate updates when the model changes.

Reply:
@Vihanth  you can set this as environment variable and use it , this is the only unresolved comment i believe